adal/src/main/java/com/microsoft/aad/adal/BrokerAccountServiceHandler.java

```diff
@@ -128,9 +128,10 @@ public Bundle getAuthToken(final Context context, final Bundle requestBundle) th
             public void onSuccess(BrokerAccountServiceConnection result) {
                 final IBrokerAccountService brokerAccountService = result.getBrokerAccountServiceProvider();
                 try {
+                    HttpWebRequest.throwIfNetworkNotAvaliable(context);
```

Comment:
Does it work? Aren't we still skipping the access token lookup even it's still valid?